deploy/backplane/lpsre/rhoc/02-rhoc-lpsre-admins-cluster.SubjectPermission.yml

```diff
@@ -0,0 +1,12 @@
+apiVersion: managed.openshift.io/v1alpha1
+kind: SubjectPermission
+metadata:
+  name: backplane-lpsre-rhoc-admins-project
```

Comment:
Rename file to "02-rhoc-lpsre-admins-project.SubjectPermission.yml" (/cluster/project/)